harbor-helm/values.yaml

```diff
@@ -609,13 +609,13 @@ trivy:
   podAnnotations: {}
 
 notary:
-  enabled: false
+  enabled: true
   server:
     # set the service account to be used, default if left empty
     serviceAccountName: ""
     image:
-      repository: goharbor/notary-server-photon
-      tag: v2.0.2
+      repository: registry.suse.com/registry/harbor-notary-server
```

Comment:
For master, we're switching to `harbor` prefix...

Reply:
Ugh, right, thanks for reminding me :+1: